core/src/main/java/io/questdb/cairo/SymbolMapReaderImpl.java

```diff
@@ -133,7 +133,7 @@ public void of(CairoConfiguration configuration, Path path, CharSequence columnN
             final long offsetMemSize = SymbolMapWriter.keyToOffset(symbolCount) + Long.SIZE;
             LOG.debug().$("offsetMem.of [columnName=").$(path).$(",offsetMemSize=").$(offsetMemSize).I$();
             this.offsetMem.of(ff, path, offsetMemSize, offsetMemSize, MemoryTag.MMAP_INDEX_READER);
-            symbolCapacity = offsetMem.getInt(SymbolMapWriter.HEADER_CAPACITY);
+            this.symbolCapacity = SymbolMapWriter.getSymbolCapacity(configuration, offsetMem);
```

Comment:
if you set reader symbol capacity to 

``` 
Math.max(configuration.getDefaultSymbolCapacity(), offsetMem.getInt(HEADER_CAPACITY))
```

when someone changes config default symbol capacity to a large number the readers will not be able to read old symbol columns